### ISSUE TITLE
add to package.json for easy install/require

### ISSUE THREAD

**Issue by bradoyler** [mymonero/mymonero-core-js#4]
For security reasons, I understand why you might not want to add this to npm registry. 

But... if we added a `package.json` it would make it much easier to use this library in a project, via:
```
$ npm i <repo url> --save
```

Then in my node or webpack project I could just do:
```
const walletUtils = require('mymonero-core-js/monero_utils/monero_wallet_utils')
```

Thoughts?

**Comment by paulshapiro** [comment]
Hi, thanks for the suggestion. It's tough. This is fine with me, but I would like to accompany such a change with ample notice about the security risk of doing so, with suggestions of best practices to mitigate the risks. Things such as forking the repo and referencing your fork from your package.json file, so that you are in control of what gets installed. Would you like to make the initial PR? I can tweak it to readiness as necessary.

**Comment by bradoyler** [comment]
Yep. Ill put something together. 👍

**Comment by bradoyler** [comment]
closed via #5

**Comment by bradoyler** [comment]
@paulshapiro Btw, this was why I was keen on doing this: https://github.com/bradoyler/xmr-paper/commit/4608041f75ab6ac977b00d324b968d1d648d5190 😸
